### DISCUSSION TITLE
I beleve i have made the words smallest deauther.....

### DISCUSSION THREAD

**Discussion by ju5t3nc4s3** [spacehuhn/esp8266_deauther#1646] [Show and tell]
Here is a type-c deauther. Can be pluged into anything. Even phone port to get power. And is still programable with a adapter throw type-c usb port.   Only part really making it  visable is the antenna.. Can be flashed with Auto-deauth-all   and run very hidden for a long time. Making it very dangerous  LoL.....
![IMG_20230429_224111](https://user-images.githubusercontent.com/79302448/235334589-cb839fc0-d923-4c6c-be34-ee4c11c3b7bd.jpg)
![IMG_20230429_224100](https://user-images.githubusercontent.com/79302448/235334592-37677ffc-39ef-4219-864c-1a3a2885e27e.jpg)

**Comment by ryaneditraj** [comment]
Nice, what chip did you use ?

**Reply by ju5t3nc4s3** [reply]
Esp8285  has the flash intergrated.

**Reply by sahaj24** [reply]
I also want to make this but how did you code that? like did you have its pinout? where did you connect c type in which pinouts?
